frontend/src/context/VaultContext.tsx

```diff
@@ -43,6 +45,10 @@ export const VaultContextProvider: React.FC = ({ children }) => {
     ? (tokenAddress: string, amount: BigNumber) => withdraw(selectedVault.id, tokenAddress, address, amount, signer)
     : null
 
+  const withdrawRewardsFromVault = address && signer && selectedGeyser
```

Comment:
it would add complexity to the type of the function though, because then the type would be `"" | null | undefined | ((receipt: TransactionReceipt) => Promise<...>)`

Reply:
ahh, you are correct! ignore this!